third_party/terraform/tests/data_source_google_compute_router_test.go

```diff
@@ -0,0 +1,51 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccDataSourceComputeRouter(t *testing.T) {
+	t.Parallel()
+	//region := getTestRegionFromEnv()
+	name := acctest.RandomWithPrefix("router-test")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceComputeRouterConfig(name),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.google_compute_router.myrouter", "id", name),
+					resource.TestCheckResourceAttr("data.google_compute_router.myrouter", "name", name),
+					resource.TestCheckResourceAttr("data.google_compute_router.myrouter", "network", fmt.Sprintf("https://www.googleapis.com/compute/v1/projects/%s/global/networks/%s", getTestProjectFromEnv(), name)),
+				),
+			},
+		},
+	})
+}
+
+func testAccDataSourceComputeRouterConfig(name string) string {
+	return fmt.Sprintf(`
+		resource "google_compute_network" "foobar" {
+			name = "%s"
+			auto_create_subnetworks = false
+		}
+		resource "google_compute_router" "foobar" {
+			name = "%s"
+			network = "${google_compute_network.foobar.name}"
+			bgp {
+				asn = 64514
+			}
```

Comment:
Spacing is mixed up here, can you format this like the `data "google_compute_router"` below?